### PR TITLE
Added options to pass args

### PR DESCRIPTION
**Pull request by niksingh710**
This will allow users to use ignorecase or other utilities

### CODE REVIEW

**Comment by morditore** [comment]
@nickclyde Can we get this merged please?

File: rofi-bluetooth [hunk 1]
```diff
@@ -305,7 +305,7 @@ show_menu() {
 }
 
 # Rofi command to pipe into, can add any options here
-rofi_command="rofi -dmenu -no-fixed-num-lines -yoffset -100 -i -p"
+rofi_command="rofi -dmenu $@"
```
Comment (morditore) [inline rofi-bluetooth:308]: It might be a good idea to keep the -p prompt argument within this script.

Comment (niksingh710) [reply rofi-bluetooth]: Performed The changes.
Check if this works.

**Review by nickclyde — APPROVED** [review]
Looks good 👍

**Pull request merged**